### PR TITLE
Add style to project linking in index

### PR DESCRIPTION
**Pull request by amadden80**
I've added some nice styles

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: lightblue;
+}
```
Comment (WilliamfShaw) [inline styles.css]: why lightblue??

File: index.html [hunk 2]
```diff
@@ -3,8 +3,9 @@
 <head>
   <meta charset="UTF-8">
   <title>Document</title>
+  <link rel="stylesheet" href="styles.css" media="screen" title="no title" charset="utf-8">
 </head>
 <body>
```
Comment (WilliamfShaw) [inline index.html]: 👾

File: styles.css (hunk 1 above)
Comment (WilliamfShaw) [inline styles.css]:
```
def madden 
 puts "cool"
end
```

Comment (amadden80) [inline styles.css]:
```
const lightblue = 'duh';
```

**Pull request merged**